packages/app/src/styles/_emoji-mart.scss

```diff
@@ -0,0 +1,19 @@
+// Override emoji-mart style
+nav.emoji-mart-anchors {
+  display: flex !important;
+}
+
+button.emoji-mart-anchors {
+  display: inline-block !important;
+  color: #858585 !important;
```

Comment:
Could you tell me the reason to set the color #858585?

if you want to change the color, please write it  on apply-colors.css not this file.
To change the color for light/ dark theme,  write on apply-colors-light.scss , apply-colors-dark.scss.

Reply:
This style is used when the emoji picker uses a modal. I will find the possibilities without using this style since we're already using the emoji picker component